src/Core/Checkout/Document/DocumentGenerator/DocumentGeneratorInterface.php

```diff
@@ -3,19 +3,13 @@
 namespace Shopware\Core\Checkout\Document\DocumentGenerator;
 
 use Shopware\Core\Checkout\Document\DocumentConfiguration;
-use Shopware\Core\Checkout\Order\OrderEntity;
 use Shopware\Core\Framework\Context;
 
 interface DocumentGeneratorInterface
 {
-    public function supports(): string;
+    public function supports(string $documentType): bool;
```

Comment:
this is a breaking change 😉

Reply:
This whole PR is a breaking change 😉

Reply:
then the target should be the major branch :blush: